src/wallet/rpcwallet.cpp

```diff
@@ -3347,6 +3348,8 @@ UniValue bumpfee(const JSONRPCRequest& request)
             "         \"UNSET\"\n"
             "         \"ECONOMICAL\"\n"
             "         \"CONSERVATIVE\"\n"
+            "     \"reduce_output\"     (numeric, optional) Increase the fee by reducing the output value of the output at\n"
```

Comment:
`reduce_output` or `reduceOutput`?

Reply:
I think the standard these days is to use snake case, even for new RPC commands. See e.g. `address_style` in `getrawtransaction` which was from a recent implementation.